ts/store/actions/messages.ts

```diff
@@ -36,7 +39,7 @@ export const loadMessages = createAsyncAction(
 )<void, ReadonlyArray<string>, Error>();
 
 export const loadMessagesCancelled = createStandardAction(
-  "MESSAGES_LOAD_CANCELLED"
+  "MESSAGES_LOAD_CANCELED"
```

Comment:
Since we are in the mood of preparatory cleaning, what you think about removing this action? Looking the code seems that the action `loadMessagesCancelled` is dispatched just one time but is never used or read.

Reply:
Good point. I went the extra mile by [removing the logic altogether](https://github.com/pagopa/io-app/pull/3407/commits/7c52ecac438133fbee8f2a272adbba0fbd226644) since there is no point for it anymore.
I was working on it on a separate branch therefore is not coming out of the blue :-) 

Tested locally and seems to be working fine.